src/EditorFeatures/Test2/Expansion/NameExpansionTests.vb

```diff
@@ -222,7 +222,7 @@ class C
             Await TestAsync(input, expected, expandParameter:=True)
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Expansion)>
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/11979"), Trait(Traits.Feature, Traits.Features.Expansion)>
```

Comment:
I will work with the IDE team to help understand why they were depending on incorrect behavior of the compiler APIs.

---
In reply to: [67028564](https://github.com/dotnet/roslyn/pull/11981#discussion_r67028564) [](ancestors = 67028564,67024193,67016763,67015720)